src/main/frontend/components/conversion.cljs

```diff
@@ -1,180 +0,0 @@
-(ns frontend.components.conversion
-  (:require [clojure.core.async :as async]
```

Comment:
Removed this since it makes it hard to move related fns to the related deps, I assume most of our users' graphs have been migrated to the new file format. 

We can notice users download the latest release that supports the migration to migrate if a lot of graphs are still use the legacy format.